halo2_backend/src/poly/commitment.rs

```diff
@@ -121,9 +121,6 @@ pub trait MSM<C: CurveAffine>: Clone + Debug + Send + Sync {
 
 /// Common multi-open prover interface for various commitment schemes
 pub trait Prover<'params, Scheme: CommitmentScheme> {
-    /// Query instance or not
-    const QUERY_INSTANCE: bool;
-
```

Comment:
I see that many updates of PR come from the removal of this feature.
Can you explain why we don't need this feature?

(Not related to main topic, but would be better if we discuss & remove feature in separate PR)